Answers.md

```diff
@@ -1,6 +1,11 @@
 ### Questions:
 1. What is the DOM?
+- Document Object Model
```

Comment:
True. Would you consider to be more specific? For example, what is the use of DOM?